### PR TITLE
Remove LP calls and use existing user positions data

### PR DESCRIPTION
**Pull request by jinchung**
Removed the batch of Infura calls we do for getting Uniswap LP data.
Instead, reuse the user positions data we already fetch from the subgraph.

We have dropped support for UNI-V1 tokens, so currently we are only surfacing UNI-V2 pools.
We can do a separate PR for UNI-V3 pool token surfacing (there are a few changes to the V3 subgraph schema we will need to account for.)

Deposit / withdraw buttons on LP pools should now redirect to the Uniswap V2 endpoint.

Fixes RNBW-1401

PoW: https://recordit.co/lkqm2h0Kh9

### CODE REVIEW

**Review by skylarbarrera — APPROVED** [review]
🔥

File: src/redux/usersPositions.ts [hunk 1]
```diff
@@ -58,14 +63,29 @@ interface Position {
   token1PriceUSD: number;
 }
 
-export type StoredPositions = Position &
+export type StoredPosition = Position &
```
Comment (brunobar79) [inline src/redux/usersPositions.ts:66]: Is it worth renaming these types to `*UniswapPosition*` assuming we will have other sources in the future?

**Review by brunobar79 — APPROVED** [review]
👸

File: src/redux/usersPositions.ts (hunk 1 above)
Comment (jinchung) [reply src/redux/usersPositions.ts]: good idea!

File: src/apollo/queries.js [hunk 2]
```diff
@@ -277,19 +277,23 @@ export const GET_BLOCKS_QUERY = timestamps => {
...
   query liquidityPositions($user: Bytes!) {
-    liquidityPositions(where: { user: $user }) {
+    liquidityPositions(where: { user: $user, liquidityTokenBalance_gt: 0 }) {
       pair {
         id
         reserve0
         reserve1
         reserveUSD
         token0 {
+          decimals
```
[the remark this answers is not in the document]
Comment (jinchung) [reply src/apollo/queries.js:287]: 😭

**Pull request merged**